booking-manager.js

```diff
@@ -1,12 +1,6 @@
 export class BookingManager {
-  /*   constructor() {
-      this.bookBtns = document.querySelectorAll('.challenges-container__challenge__button');
-      this.bookBtns.forEach((bookingBtn) => {
-        bookingBtn.addEventListener('click', () => this.createBookingPage());
-      });
-    } */
 
-  // Method for creating elements
+  // Method for creating elements with fewer lines
```

Comment:
This comment says "with fewer lines", making me wonder, fewer than what? Fewer than it used to be before? At what point?

I think what I'm seeing here is an attempt to use code comments as a way to tell a story about the code. I think that's a risky proposition. In fact, comments are generally "bad". Comments should only exist when code is difficult to read, and code should ideally by easy to read, meaning that comments are usually an indicator that code should be simplified. Of course that's not always possible, so comments are sometimes needed. 

But what's much worse than having no comments, is having misleading comments, and that can sometimes happen. Here's what I wrote on the PR of a classmate of yours, who had accidentally moved a line of code without moving the associated comment (which is not a beginner's mistake, but actually very common).

> My theory is that most programmers stop reading comments, unless they don't understand the code. The comments are colored differently and programmers, being lazy, just filter them out to have to read less. This makes sense, but it does mean that when you make changes to code, it's very easy to forget updating comments to reflect those changes.
> 
> For that reason, it's usually a good idea to avoid having comments, and instead, writing the code in a way that makes it easily readable, using good variable names etc. 

In this case, your function `createElement()` has a fine name that already tells me everything that the comment is telling me (it's a function, for creating elements).

The additional "with fewer lines" is not relevant (nor does it make sense) to someone reading the code at a future point in time, so it risks being confusing instead of helpful.